erpnext/selling/doctype/quoting_sheet/quoting_sheet.json

```diff
@@ -155,12 +137,6 @@
    "options": "Quoting Sheet Item",
```

Comment:
- Is the `Customer Provided Item` position change pushed in this PR ?

Reply:
Yes